Engine/ac/screenoverlay.cpp

```diff
@@ -38,6 +38,22 @@ void ScreenOverlay::ReadFromFile(Stream *in, int32_t cmp_ver)
         _offsetX = in->ReadInt32();
         _offsetY = in->ReadInt32();
     }
+    if (cmp_ver >= 2)
```

Comment:
Please give new save format a much higher number in case there will be change in 3.* branch